pages/coq-team.html

```diff
@@ -0,0 +1,503 @@
+    <div class="name"><a href="https://mattam.org">Matthieu Sozeau</a></div><div class="nickname">mattam82</div>
+    <div class="position">Inria Researcher in the Gallinette Team, Nantes, France</div>
+    <div class="role">Co-RM: 8.2-8.5</div>
+    <div class="components">General Maintenance, opam, Theory, Kernel, Elaboration, Unification, Typeclasses, Program, Universes, Equations, MetaCoq</div>
+  </div>
+</div>
+<div class="team">
+  <div class="team-name"><h1>Core Team</h1></div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/YvesBertot.png" alt="Yves Bertot" width="200px"/></div>
+    <div class="name"><a href="http://www-sop.inria.fr/members/Yves.Bertot/research.html">Yves Bertot</a></div><div class="nickname">YvesBertot</div>
+    <div class="position">Inria Researcher, Head of the Stamp Team, Sophia-Antipolis, France</div>
+    <div class="role">Head of the Coq Consortium</div>
+    <div class="components">Coq'Art co-author, Function, Plugin-Tutorial, Documentation, 
+      Mathematical Components</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/maximedenes.jpg" alt="Maxime Dénès" width="200px"/><br/></div>
+    <div class="name"><a href="https://maximedenes.fr">Maxime Dénès</a></div>
+    <div class="nickname">maximedenes</div>
+    <div class="position">Inria Engineer in the Coq Consortium, attached to the Stamp Team, Sophia-Antipolis, France</div>
+    <div class="role">RM: 8.5, 8.6, 8.7, 8.8, 8.13</div>
+    <div class="components">General Maintenance, CI, <verb>native_compute</verb>, Kernel, Unification, VSCoq</div>
+  </div>  
+  <div class="person">
+    <div class="picture"><img src="/files/team/ejgallego.jpg" alt="Emilio Jesús Gallego Arias" width="220px"/></div>
+    <div class="name"><a href="https://www.irif.fr/~gallego/">Emilio Jesús Gallego Arias</a></div><div class="nickname">ejgallego</div>
+    <div class="position">Inria Starting Researcher in the pi.r2 Team, Paris, France</div>
+    <div class="role">RM: 8.12</div>
+    <div class="components">General Maintenance, Interfaces, Build System (dune), SerAPI, JSCoq</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/SkySkimmer.png" alt="Gaëtan Gilbert" width="200px"/><br/></div>
+    <div class="name"><a href="https://www.ls2n.fr/annuaire/Gaetan%20GILBERT/">Gaëtan Gilbert</a></div>
+    <div class="nickname">SkySkimmer</div>
+    <div class="position">Inria Engineer in the Coq Consortium, attached to the Gallinette Team, Nantes, France</div>
+    <div class="components">General Maintenance, Theory, CI, Kernel, Universes, SProp, Lean Importer</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/JasonGross.jpg" alt="Jason Gross" width="150px"/></div>
+    <div class="name"><a href="https://people.csail.mit.edu/jgross/">Jason Gross</a></div><div class="nickname">JasonGross</div>
+    <div class="position">PhD Student at MIT, MA, USA</div>
```

Comment:
```suggestion
    <div class="position">MIT PhD, MA, USA</div>
```
No longer a student (though my future plans are still in flux)

Reply:
Done